internal/wire/wire_test.go

```diff
@@ -73,26 +69,14 @@ func runTest(m *testing.M) int {
 func loadTestData(context interface{}, db *db.DB) error {
 
 	// -----------------------------------------------------------
-	// Load example Items.
-
-	items, err := itemfix.Get()
-	if err != nil {
-		return err
-	}
-
-	if err := itemfix.Add(context, db, items); err != nil {
-		return err
-	}
-
-	// -----------------------------------------------------------
-	// Load example relationships, views, and patterns.
+	// Load example items, relationships, views, and patterns.
 
-	rels, vs, pats, err := wirefix.Get()
+	items, rels, vs, pats, err := wirefix.Get()
```

Comment:
I think we were using `itm` for singular cases, because that interferes with our `item` package.